HACKING.md

```diff
@@ -450,45 +513,50 @@ with `gunzip` and placed into `~/.spread/qemu` as above.
 
 #### Running spread with QEMU
 
-Finally, you can run the spread tests for Ubuntu 16.04 LTS 64-bit with:
+Finally, you can run the spread tests for Ubuntu 18.04 LTS 64-bit with:
 
-    $ spread -v qemu:ubuntu-16.04-64
+    $ spread -v qemu:ubuntu-18.04-64
 
-To run for a different system, replace `ubuntu-16.04-64` with a different system
-name.
+>To run for a different system, replace `ubuntu-18.04-64` with a different system
+name, which should be a basename of the [built](#building-spread-vm-images) or 
+[downloaded](#downloading-spread-vm-images) Ubuntu image file.
 
 For quick reuse you can use:
 
-    $ spread -reuse qemu:ubuntu-16.04-64
+    $ spread -reuse qemu:ubuntu-18.04-64
 
 It will print how to reuse the systems. Make sure to use
 `export REUSE_PROJECT=1` in your environment too.
 
 #### Running UC20 spread with QEMU
 
+<!-- TODO: (B) Is it still working this way? -->
```

Comment:
it would be great if you could update this section, indeed some things have landed upstream so it is not needed to build from the PR mentioned anymore

Reply:
Great. Thank you. @anonymouse64, from whom can I get the details to understand what is the updated way of doing it?

Reply:
It looks like the updated way to do this is actually just to install ovmf and using spread from upstream AFAICT, so I would recommend just trying to install spread and then run some spread tests locally and see if it works

Reply:
Great. Thank you. I'll try this one.